mitmproxy/flow/export.py

```diff
@@ -29,7 +40,7 @@ def curl_command(flow):
     data += "'%s'" % full_url
 
     if flow.request.content:
-        data += " --data-binary '%s'" % flow.request.content
+        data += " --data-binary '%s'" % _native(flow.request.content)
```

Comment:
I'm all but an expert on that, but doesn't that look like a `bytes_to_escaped_str` use case?